### ISSUE TITLE
You need to have Ruby and Sass installed and in your PATH for this task to work.

### ISSUE THREAD

**Issue by ivxpingg** [gruntjs/grunt-contrib-sass#229]
![image](https://cloud.githubusercontent.com/assets/6150766/17482270/9727247e-5db3-11e6-86a9-d42ebe367489.png)

**Comment by ivxpingg** [comment]
I have installed ruby and sass.

**Comment by guvenckardas** [comment]
+1

**Comment by jmanlapid** [comment]
+1

**Comment by badoubadou** [comment]
+ 1

**Comment by jaspersorrio** [comment]
+1

**Comment by yinix** [comment]
+1

**Comment by brainstrategist** [comment]
Same issue when the command was prompt by a php script from a git webhook. It's weird because it's working well when the command is running through the cli

**Comment by juanvillegas** [comment]
any ideas?

**Comment by FloFzks** [comment]
+1

**Comment by atoner595** [comment]
+1

**Comment by vinimdocarmo** [comment]
+1

**Comment by juanvillegas** [comment]
I finally fixed mine by resolving a node conflict. Basically, I had two versions of node, one I installed manually and the other installed using brew. So maybe you should check that.

**Comment by aszmyd** [comment]
This package works nice without Ruby installed on machine: https://github.com/sindresorhus/grunt-sass

**Comment by 5ika** [comment]
grunt-sass documentation is not very clear. To avoid the need of Ruby, you can try this:
```bash
npm uninstall --save grunt-contrib-sass
npm install --save node-sass grunt-sass
```

**Comment by mattsince87** [comment]
you need to install Sass `gem install sass` or `sudo gem install sass` if like me you're on Mac

**Comment by honsa** [comment]
thx @5ika that was the problem. Not sure where i got the wrong line to install ruby gems

**Comment by sdoasddd0920** [comment]
+1

**Comment by prakyathacharya** [comment]
watch this instead

https://www.youtube.com/watch?v=VSQf10i0_7c

**Comment by strider72** [comment]
mattsince87 nailed it.  You have to install the Sass gem in Ruby

#issuecomment-301067967

**Comment by LorenaGitHub** [comment]
![image](https://user-images.githubusercontent.com/48281904/54130214-44720780-4410-11e9-8f6a-13212d2e61ef.png)

I have the same problem, I have installed ruby ​​and the sass module and it does not work for me.

**Comment by LorenaGitHub** [comment]
![image](https://user-images.githubusercontent.com/48281904/54130558-e691ef80-4410-11e9-8e85-2cf3b2bc915d.png)
this is my configuration file for Gruntfile.js

**Comment by Therealskythe** [comment]
To everyone saying to install the Ruby SASS gem:
**Ruby SASS is deprecated** (since March 2019) and users are told to 'migrate to another implementation'.

Problem is, even after installing node-sass globally the error message posted above still appears. And nobody seems to know a way out.

**Comment by jowe1** [comment]
What I needed for it to work was:
sudo apt install ruby-sass

Running Ubuntu 18.04. I don't know why this isn't mentioned in ANY tutorial / help / guide, but I really needed to install the ruby-sass package:
jwetzel@uldren:~/p/Lightningsoul.com$ sass --version
Sass 3.4.23 (Selective Steve)

Just check if you have installed sass, if not it will say:
jwetzel@uldren:~/p/Lightningsoul.com$ sass --version

Command 'sass' not found, but can be installed with:

sudo apt install ruby-sass

Hope this helps!

**Comment by silviuburceadev** [comment]
I have found another solution:
`npm install --save-dev sass`

You can't run `grunt` directly because `grunt-contrib-sass` will still complain about missing `sass.` However, if you wrap it in a npm script, e.g. in your package.json scripts section: `"build": "grunt build"` (assuming build task will invoke sass compilation), you can run `npm run build`. This works because npm scripts will export node modules that provide binaries to the PATH, including `sass`, so `grunt-contrib-sass` will not complain anymore.

tl;dr install `sass` and use a npm script to invoke grunt. Or just `npm install -g sass` and run grunt the hell you want.

**Comment by torweb-net** [comment]
> You have 2 options, both without Ruby:
> 
> 1. Sass as dev dependency `npm install --save-dev sass`
>    Invoke grunt from a npm script, since this will allow npm to add to PATH the node modules that expose binaries, like sass.
> 2. Sass installed globally `npm install -g sass`
>    Invoke grunt whatever you like.

This worked!

**Comment by odil-io** [comment]
`sudo gem install sass` did the trick for me on OSX 12.0.1.

**Comment by huynhthienthach** [comment]
> You have 2 options, both without Ruby:
> 
> 1. Sass as dev dependency `npm install --save-dev sass`
>    Invoke grunt from a npm script, since this will allow npm to add to PATH the node modules that expose binaries, like sass.
> 2. Sass installed globally `npm install -g sass`
>    Invoke grunt whatever you like.

this worked!!!

**Comment by jelly823** [comment]
`npm install -g ruby sass`

![image](https://user-images.githubusercontent.com/31650787/187911440-90c28106-dc31-4645-9365-a5474f295f81.png)
